package.json

```diff
@@ -2,7 +2,7 @@
   "name": "cloud-file-manager",
   "description": "Wrapper for providing file management for web applications",
   "author": "The Concord Consortium",
-  "version": "1.4.0",
+  "version": "1.4.1",
   "repository": {
     "type": "git",
     "url": "https://github.com/concord-consortium/cloud-file-manger"
```

Comment:
Not needed for this PR but I just noticed the repository url has a typo:

`"url": "https://github.com/concord-consortium/cloud-file-manger"`

should be

`"url": "https://github.com/concord-consortium/cloud-file-manager"`

Reply:
I added a commit to fix the typo as long as I'm in the neighborhood.